api/tests/Api/CampCollaborations/UpdateCampCollaborationTest.php

```diff
@@ -265,6 +265,53 @@ public function testPatchStatusFromInactiveToInvitedSendsInviteEmail() {
         $this->assertEmailCount(1);
     }
 
+    public function testPatchCampCollaborationValidatesInvalidColor() {
```

Comment:
Thank you very much for the update tests
is it also allowed to send these properties on create?

Reply:
Yes. But the CreateProfileTest was empty, so I skipped adding them there.